tests/reconfig/results/v1.0.0.md

```diff
@@ -0,0 +1,42 @@
+# Reconfiguration testing Results
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.4-gke.900
+- Zone: europe-west2-b
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+NGF deployment:
+
+- NGF version: edge - git commit 72b6c6ef8915c697626eeab88fdb6a3ce15b8da0
+- NGINX Version: 1.25.2
+
+## Results Table
+
+| Test number | NumResources | TimeToReadyTotal | TimeToReadyAvgSingle | NGINX reloads (total) | NGINX reload avg time (ms) |
+| ----------- | ------------ | ---------------- | -------------------- | --------------------- | -------------------------- |
+| 1           | 30           | 5                | 5                    | 1 (2)                 | 166                        |
```

Comment:
I know there is a formula in the other doc. However, would it possible to include explanation of total number of resources? to avoid performing calculations on the fly


For example, after the table:

30 NumResources = 1 Gateway + 1 ... +  k * Pods + l * HTTPRoutes 
150 NumResources = ....